.rubocop.yml

```diff
@@ -0,0 +1,11 @@
+AllCops:
+  RunRailsCops: true
+  Exclude:
+    - db/schema.rb
```

Comment:
It is simply not needed. While updating db in `dummy` project from the spec `spec/dummy/db/schema.rb` is updated no `db/schema.rb`